openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -41,28 +41,6 @@ function download {
   done
 }
 
-function download_ingress {
```

Comment:
I'm guessing this ain't intentional?

Reply:
yes, that's a rebase issue - I anm aware of that ;-)